lib/fromZodError.ts

```diff
@@ -0,0 +1,59 @@
+import * as zod from 'zod';
+
+import {
+  ISSUE_SEPARATOR,
+  MAX_ISSUES_IN_MESSAGE,
+  PREFIX,
+  PREFIX_SEPARATOR,
+  UNION_SEPARATOR,
+} from './config';
+import {
+  conditionallyPrefixMessage,
+  FromZodIssueOptions,
+  getMessageFromZodIssue,
+} from './fromZodIssue';
+import { ValidationError } from './ValidationError';
+
+export type ZodError = zod.ZodError;
+
+export type FromZodErrorOptions = FromZodIssueOptions & {
+  maxIssuesInMessage?: number;
+};
+
+export function fromZodError(
+  zodError: ZodError,
+  options: FromZodErrorOptions = {}
+): ValidationError {
+  const {
+    maxIssuesInMessage = MAX_ISSUES_IN_MESSAGE,
+    issueSeparator = ISSUE_SEPARATOR,
+    unionSeparator = UNION_SEPARATOR,
+    prefixSeparator = PREFIX_SEPARATOR,
+    prefix = PREFIX,
+    includePath = true,
+  } = options;
+
+  const zodIssues = zodError.errors || [];
```

Comment:
[question] Are we doing the fallback check after all? i.e `|| []`

Reply:
No, this is a mistake. Thanks for pointing it out.